mobile-app/app/styles.json

```diff
@@ -8034,6 +8034,10 @@
 	"text-justify": {
 		"textAlign": "justify"
 	},
+	"text-xxs": {
```

Comment:
Not sure if it's okay to add here @thedoublejay

Reply:
styles.json is generated. To extend tailwind classes, modify `mobile-app/app/tailwind.config.js` instead

Then in `mobile-app/app` run `npx create-tailwind-rn`